### ISSUE TITLE
generated barcode is incorrect

### ISSUE THREAD

**Issue by hosamre94** [Tagliatti/NetBarcode#21]
I'm trying to generate a barcode based on the code below,

`var barcode = new Barcode("G7G95FTU");
 var barcodeBytes = barcode.GetByteArray();`

but here is the barcode that's generated by the library, and when I scanned it with a barcode scanner I got this ("**G7GFTU**")

![image](https://user-images.githubusercontent.com/15855930/122012550-f9e94d80-cdc5-11eb-95bc-1353bbf0219c.png)

**Comment by Tagliatti** [comment]
Hi, there really is a bug in Code128's algorithm with this string. In my tests I couldn't even read the barcode. But you can use Code39E which is working correctly for your example until it is fixed.
Thank you very much for reporting the bug. 

`var barcode = new Barcode("G7G95FTU", NetBarcode.Type.Code39E, true);`

**Comment by DevTown** [comment]
Is the bug in all 128 barcodes or only in the base ?

**Comment by Tagliatti** [comment]
@DevTown Thank you very much for asking, I did some tests now and found out that the problem is in Code128's automatic mode switching. If you force Code128A, the example works. 

```

// Not work
var barcode = new Barcode("G7G95FTU", NetBarcode.Type.Code128, true);
// Work
var barcode = new Barcode("G7G95FTU", NetBarcode.Type.Code128A, true);
// Work
var barcode = new Barcode("G7G95FTU", NetBarcode.Type.Code128B, true);
```

**Comment by DevTown** [comment]
Thanks for the fast answer . Will check my code for this tomorrow.

**Comment by Tagliatti** [comment]
@hosamre94 @DevTown Bug fixed in version 1.4.3
